src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -131,17 +156,46 @@ public void configure(Map<String, ?> configs) {
         storageQuotaHard = config.getHardStorageQuota();
         storageCheckInterval = config.getStorageCheckInterval();
         logDirs = Arrays.stream(config.getLogDirs().split(",")).map(Paths::get).collect(Collectors.toList());
+        thresholdMap = new HashMap<>();
+        String percents = config.getStoragePercents();
+        String[] split = percents.split("\\|");
+        for (String s : split) {
+            String[] inner = s.split(":");
+            Path path = Paths.get(inner[0]);
+            if (!logDirs.contains(path)) {
+                throw new IllegalArgumentException(StaticQuotaConfig.STORAGE_QUOTA_PERCENTAGES_PROP + " property contains path that's not part of logDirs: " + path + " - " + logDirs);
+            }
+            double soft = Double.parseDouble(inner[1]);
+            double hard = Double.parseDouble(inner[2]);
```

Comment:
Can we add a bounds check on there percentages?

Reply:
What's a bounds check?
That `soft` is `<` than `hard`?

Reply:
0 < soft < hard <= 100